frontend/src/pages/_app.tsx

```diff
@@ -56,12 +55,6 @@ export default function MyApp({ Component, pageProps }) {
         document.body.style.setProperty("--code-line-height", codeLineHeight)
     }, [codeLineHeight])
 
-    useEffect(() => {
-        if (isMacOS()) {
-            document.body.classList.add("device-macos")
-        }
-    }, [])
```

Comment:
hm why?

Reply:
this class has since been removed from our css, and I wanted to clean up dead code